src/redux/categories/categories.js

```diff
@@ -1,10 +1,10 @@
 import types from '../types/types';
 
 const initialState = {
-  categories: [],
+  categories: '',
 };
 
-export const checkCategoryStatus = (id) => ({ type: types.STATUS_CHECKED, id });
+export const checkCategoryStatus = () => ({ type: types.STATUS_CHECKED });
 
 export default function categoryReducer(state = initialState, action) {
   switch (action.type) {
```

Comment:
- Please, kindly update the categories to return the 'Under construction' text as your's is returning an empty string. see below ::
```js
export default function categoryReducer(state = initialState, action) {
  switch (action.type) {
    case types.STATUS_CHECKED:
      return {
        ...state,
        categories: 'Under construction',
      };
    default:
      return state;
  }
}
```

Reply:
Thanks for the observation. I will update accordingly